### ISSUE TITLE
Error saving checkpoint

### ISSUE THREAD

**Issue by TjebbeVQ** [xinyadu/nqg#28]
tjebbe@tjebbe-VirtualBox:~/nqg/sentence$ th train.lua -config config-trainLoading data from 'data/qg-train.t7'...	
 * vocabulary size: source = 45004; target = 28003	
 * additional features: source = 0; target = 0	
 * maximum sequence length: source = 100; target = 50	
 * number of training sentences: 69979	
 * maximum batch size: 64	
Building model...	
 * using input feeding	
Initializing parameters...	
 * number of parameters: 26319403	
Preparing memory optimization...	
 * sharing 71% of output/gradInput tensors memory between clones	
Start training...	
	
Epoch 1 ; Iteration 50/1140 ; Learning rate 1.0000 ; Source tokens/s 171 ; Perplexity 72871.79	
Epoch 1 ; Iteration 100/1140 ; Learning rate 1.0000 ; Source tokens/s 175 ; Perplexity 27607.56	
Epoch 1 ; Iteration 150/1140 ; Learning rate 1.0000 ; Source tokens/s 187 ; Perplexity 12133.82	
Epoch 1 ; Iteration 200/1140 ; Learning rate 1.0000 ; Source tokens/s 181 ; Perplexity 6763.18	
Epoch 1 ; Iteration 250/1140 ; Learning rate 1.0000 ; Source tokens/s 182 ; Perplexity 4378.42	
Epoch 1 ; Iteration 300/1140 ; Learning rate 1.0000 ; Source tokens/s 183 ; Perplexity 3127.49	
Epoch 1 ; Iteration 350/1140 ; Learning rate 1.0000 ; Source tokens/s 182 ; Perplexity 2385.68	
Epoch 1 ; Iteration 400/1140 ; Learning rate 1.0000 ; Source tokens/s 181 ; Perplexity 1878.88	
Epoch 1 ; Iteration 450/1140 ; Learning rate 1.0000 ; Source tokens/s 181 ; Perplexity 1532.97	
Epoch 1 ; Iteration 500/1140 ; Learning rate 1.0000 ; Source tokens/s 180 ; Perplexity 1294.61	
Epoch 1 ; Iteration 550/1140 ; Learning rate 1.0000 ; Source tokens/s 177 ; Perplexity 1122.80	
Epoch 1 ; Iteration 600/1140 ; Learning rate 1.0000 ; Source tokens/s 175 ; Perplexity 984.16	
Epoch 1 ; Iteration 650/1140 ; Learning rate 1.0000 ; Source tokens/s 173 ; Perplexity 874.13	
Epoch 1 ; Iteration 700/1140 ; Learning rate 1.0000 ; Source tokens/s 173 ; Perplexity 789.38	
Epoch 1 ; Iteration 750/1140 ; Learning rate 1.0000 ; Source tokens/s 174 ; Perplexity 718.86	
Epoch 1 ; Iteration 800/1140 ; Learning rate 1.0000 ; Source tokens/s 174 ; Perplexity 661.67	
Epoch 1 ; Iteration 850/1140 ; Learning rate 1.0000 ; Source tokens/s 175 ; Perplexity 612.37	
Epoch 1 ; Iteration 900/1140 ; Learning rate 1.0000 ; Source tokens/s 175 ; Perplexity 572.65	
Epoch 1 ; Iteration 950/1140 ; Learning rate 1.0000 ; Source tokens/s 177 ; Perplexity 536.55	
Epoch 1 ; Iteration 1000/1140 ; Learning rate 1.0000 ; Source tokens/s 176 ; Perplexity 504.60	
Epoch 1 ; Iteration 1050/1140 ; Learning rate 1.0000 ; Source tokens/s 177 ; Perplexity 476.61	
Epoch 1 ; Iteration 1100/1140 ; Learning rate 1.0000 ; Source tokens/s 178 ; Perplexity 449.97	
Validation perplexity: 119.12233583123	
Saving checkpoint to 'model/840B.300d.600rnn_epoch1_119.12.t7'...	
/home/tjebbe/torch/install/bin/luajit: cannot open <model/840B.300d.600rnn_epoch1_119.12.t7> in mode  w at /home/tjebbe/torch/pkg/torch/lib/TH/THDiskFile.c:673
stack traceback:
	[C]: at 0x7f248d4ac210
	[C]: in function 'DiskFile'
	/home/tjebbe/torch/install/share/lua/5.1/torch/File.lua:385: in function 'save'
	./onmt/train/Checkpoint.lua:28: in function 'save'
	./onmt/train/Checkpoint.lua:63: in function 'saveEpoch'
	train.lua:415: in function 'trainModel'
	train.lua:540: in function 'main'
	train.lua:543: in main chunk
	[C]: in function 'dofile'
	...ebbe/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk

I did set it to just 1 epoch for testing (especially for errors after training so i dont waste my time as i dont have GPU). The embeddings file i used is 6B.100d instead of 840B.300d but that shouldn't be a problem. It is the sentence part and not the paragraph (haven't tested that one).

**Comment by wentsun12** [comment]
Did you add model dir first? I'm not sure if it is the reason because mine worked fine.

**Comment by TjebbeVQ** [comment]
yeah, after i added the folder everything works smoothly

**Comment by leesurelye** [comment]
how to add model dir ?

**Comment by leesurelye** [comment]
in folder "paragraph"  ,    mkdir model
